targets/alert-cli/src/openRepo.js

```diff
@@ -29,9 +29,29 @@ export async function openRepo({ repository }) {
     //   { bare: 1, checkoutBranch: "master", fetchOpts: { prune: 1 } }
     // );
     // we use bare repo as they weight less and we only need to read commit info
-    await exec(
-      `git clone --bare https://github.com/${org}/${repositoryName} ${localPath}`
-    );
+
+    /**
+     * The option --shallow-exclude ${tag}
+     * allow us to only clone repo with history limited to a given revision (or tag)
+     * that way we do not clone the entire history nor fetch all the existing tag
+     */
+    const command = `git clone --shallow-exclude ${tag} --bare https://github.com/${org}/${repositoryName} ${localPath}`;
+    console.log(command);
+    await exec(command);
+    /**
+     * Since swallow exlude remove our base tag, we need to add it
```

Comment:
I’m having troubles understanding that one: basically we get the repo at one tag, which is excluded, and then we get that tag with deepen ? Nice and straightforward but I find the comments slightly confusing.
Also, is it possible that the previous commit won’t be the one with the tag at some point ?
Could we simply compute the previous tag directly shallow-exclude on it so we don’t have to `deepen` ? (beware the 0.1.0 corner case)

Reply:
the script only run between versions, and version are released only if data is updated.
so even if previous commit is a not a release, it's not a problem because we know that data had been modified between that commit and the next release